File: Playlist#2/AdminTokoFD/app/src/main/java/com/example/admintokofd/MainActivity.java
package com.example.admintokofd;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import com.example.admintokofd.ListPelangganActivity;

import androidx.appcompat.app.AppCompatActivity;

import com.google.firebase.auth.FirebaseAuth;

public class MainActivity extends AppCompatActivity {

    private Button btnKelolaProduk, btnKelolaPelanggan, btnLogout;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnKelolaProduk = findViewById(R.id.btnKelolaProduk);
        btnKelolaPelanggan = findViewById(R.id.btnKelolaPelanggan);
        btnLogout = findViewById(R.id.btnLogout);

        btnKelolaProduk.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, ProdukActivity.class);
            startActivity(intent);
        });

        btnKelolaPelanggan.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, ListPelangganActivity.class);
            startActivity(intent);
        });

        btnLogout.setOnClickListener(v -> {
            FirebaseAuth.getInstance().signOut();
            startActivity(new Intent(MainActivity.this, LoginActivity.class));
            finish();
        });
    }
}

File: Playlist#2/Blogging/app/src/main/java/com/example/blogging/MainActivity.kt
package com.example.blogging

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Toast
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import androidx.fragment.app.Fragment
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.example.blogapp.model.Blog
import com.example.blogging.activities.CreateBlogActivity
import com.example.blogging.adapter.BlogAdapter
import com.google.android.material.bottomnavigation.BottomNavigationView
import com.google.android.material.floatingactionbutton.FloatingActionButton
import com.google.firebase.database.FirebaseDatabase

class MainActivity : AppCompatActivity() {
    private lateinit var fabAddBlog: FloatingActionButton

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        fabAddBlog = findViewById(R.id.fabAddBlog)
        val bottomNav = findViewById<BottomNavigationView>(R.id.bottomNavigationView)

        loadFragment(HomeFragment()) // default

        bottomNav.setOnItemSelectedListener { item ->
            when (item.itemId) {
                R.id.menu_home -> {
                    fabAddBlog.visibility = View.GONE
                    loadFragment(HomeFragment())
                    true
                }
                R.id.menu_profile -> {
                    fabAddBlog.visibility = View.VISIBLE
                    loadFragment(ProfileFragment())
                    true
                }
                else -> false
            }
        }

        fabAddBlog.setOnClickListener {
            startActivity(Intent(this, CreateBlogActivity::class.java))
        }
    }

    private fun loadFragment(fragment: Fragment) {
        supportFragmentManager.beginTransaction()
            .replace(R.id.fragment_container, fragment)
            .commit()
    }
}
